bench/runbench.py

```diff
@@ -114,8 +114,8 @@ def build_rho(gitref, args, jit, build=True):
         # Clean and switch to the selected revision.
         subprocess.call(['git', 'reset', '--hard', gitref])
         # Build with JIT enabled.
-        subprocess.call(['git', 'clean', '-fd', 'HEAD'])
-        subprocess.call(['git', 'clean', '-fX', 'HEAD'])
+        subprocess.call(['git', 'clean', '-fd'])
+        subprocess.call(['git', 'clean', '-fX'])
```

Comment:
A better approach would be to do the build in a separate directory from the source directory, in which case you could just delete the entire build directory.